streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBSessionStoreSupplierTest.java

```diff
@@ -110,6 +111,7 @@ public void shouldNotBeLoggingEnabledStoreWhenLoggingNotEnabled() throws Excepti
         store.init(context, store);
         store.put(new Windowed<>("a", new SessionWindow(0, 10)), "b");
         assertTrue(logged.isEmpty());
+        context.close();
```

Comment:
you should wrap in `try{...}finally{..}` so `context.close()` gets called when the test fails